packages/admin/admin/src/index.ts

```diff
@@ -73,7 +73,7 @@ export { FinalFormSaveSplitButton } from "./FinalFormSaveSplitButton";
 export { FinalFormAutocomplete, FinalFormAutocompleteProps } from "./form/Autocomplete";
 export { FinalFormCheckbox, FinalFormCheckboxProps } from "./form/Checkbox";
 export { Field, FieldProps } from "./form/Field";
-export { FieldContainer, FieldContainerClassKey, FieldContainerComponent, FieldContainerProps } from "./form/FieldContainer";
```

Comment:
This is a breaking change, please add a changeset explaining what to do instead

Reply:
`FieldContainerComponent` should never have been exported and was likely never actually used, since it's the "uncomplete" version of `FieldContainer`... Should I add a changeset anyway?